### PR TITLE
feat(content): #222 add bookmarks pagination

### LINKED ISSUE #222: Add pagination UI
Blocked on https://github.com/mozilla/activity-streams/issues/198 for implementation

We just need some UI for loading more stuff/next on the Timeline

### PR DESCRIPTION
**Pull request by k88hudson**
Fixes #222

### CODE REVIEW

File: content-src/components/TimelinePage/TimelineBookmarks.js [hunk 1]
```diff
@@ -3,13 +3,25 @@ const {connect} = require("react-redux");
...
+const LoadMore = require("components/LoadMore/LoadMore");
 
 const TimelineBookmarks = React.createClass({
+  getMore() {
+    const bookmarks = this.props.Bookmarks.rows;
+    if (!bookmarks.length) {
+      return;
+    }
+    const beforeDate = bookmarks[bookmarks.length - 1].lastModified;
+    this.props.dispatch(RequestMoreBookmarks(beforeDate));
```
Comment (k88hudson) [inline content-src/components/TimelinePage/TimelineBookmarks.js]: This dispatches a `RECENT_BOOKMARKS_REQUEST` with the beforeDate as a query option

File: content-src/components/TimelinePage/TimelineBookmarks.js [hunk 2]
```diff
@@ -3,13 +3,25 @@ const {connect} = require("react-redux");
...
+  },
   render() {
     const props = this.props;
     return (<div className="wrapper">
       <Spotlight sites={props.Spotlight.rows} />
       <GroupedActivityFeed title="Just now" sites={props.Bookmarks.rows} length={20} />
+      <LoadMore loading={props.Bookmarks.isLoading} hidden={!props.Bookmarks.canLoadMore || !props.Bookmarks.rows.length} onClick={this.getMore}
```
Comment (k88hudson) [inline content-src/components/TimelinePage/TimelineBookmarks.js]: This shows a loading spinner/button to load more

**Comment by k88hudson** [comment]
R? @oyiptong  @pdehaan or @mzhilyaev

File: content-src/actions/action-manager.js [hunk 3]
```diff
@@ -82,7 +89,7 @@ function RequestRecentLinks(options) {
 }
 
 function RequestMoreRecentLinks(beforeDate) {
-  return RequestExpect("RECENT_LINKS_REQUEST", "RECENT_LINKS_RESPONSE", {
+  return RequestRecentLinks({
```
Comment (oyiptong) [inline content-src/actions/action-manager.js]: nit: this change isn't concerned with bookmarks and should probably be in a separate commit

**Comment by oyiptong** [comment]
r+ with a nit
